pkg/build/android_linux.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
```

Comment:
I don't know why it works fine for `linux_linux.go` without it, but it seems that it'd be nice to put here

```
//go:build linux
// +build linux
```

Reply:
Files ending with _GOOS.go or _GOARCH.go are only compiled for that OS/ARCH. That's better than build tags.

Reply:
Ok, thanks for the explanation :)